addons/website_event/models/event_tag_category.py

```diff
@@ -10,3 +10,12 @@ class EventTagCategory(models.Model):
 
     def _default_is_published(self):
         return True
+
+    website_id = fields.Many2one(comodel_name='website', string='website_id')
```

Comment:
I think we should then change the mixin used to control the publish state of tag category model. If you look at 'website/models/mixins.py', you can find that ``website.published.multi.mixin`` exists and does two main things: adding the publish mixin (publish control) and the multi-website mixin (adding the website_id field). That way we are more standard in the way we define the publish control on models.